app/src/main/java/app/insti/fragment/FileComplaintFragment.java

```diff
@@ -130,6 +130,7 @@
     private ImageButton imageButtonAddTags;
     private Button buttonAnalysis;
     private ImageButton imageActionButton;
+    protected List<Venter.TagUri> tagsApiList = new ArrayList<>();
```

Comment:
I believe `tagsApiList` can be converted to a local variable since it isn't really being used elsewhere, right?

Reply:
Yes, we can make it a local variable

Reply:
> @simranmadhok @pulsejet  I'm not familiar with the call flow of this fragment, so are you positive that the fact that the `prepareTags` call is now async wouldn't cause any issues?

What exactly do you mean by this? Do you foresee any issues with the implementation?

Reply:
Basically earlier it was guaranteed that this call would be completed _before_ the tags are requested, but now they're not, so it's potentially a NPE

Reply:
Corrections have been made. Please review